src/Symfony/Component/Messenger/Bridge/AmazonSqs/Transport/Connection.php

```diff
@@ -89,6 +90,7 @@ public function __destruct()
      * * account: identifier of the AWS account
      * * access_key: AWS access key
      * * secret_key: AWS secret key
+     * * session_token: AWS session token (required only when using temporary credentials)
```

Comment:
IMHO this sentence `(required only when using temporary credentials)` is not needed.

Reply:
alright, sorry for the bad suggestion

Reply:
ho.,. sorry, I didn't noticed it was you. Sorry for the noise